manifest_staging/charts/gatekeeper/templates/gatekeeper-audit-deployment.yaml

```diff
@@ -97,6 +97,7 @@ spec:
         {{- toYaml .Values.image.pullSecrets | nindent 8 }}
       nodeSelector:
         {{- toYaml .Values.audit.nodeSelector | nindent 8 }}
+      priorityClassName: {{ .Values.audit.priorityClassName }}
```

Comment:
do you need to add `audit.priorityClassName` and `controllerManager.priorityClassName` to helmify static values?

Reply:
I believe those are already in `cmd/build/helmify/static/values.yaml`.